devicesetup/src/main/res/layout-xlarge/activity_connecting.xml

```diff
@@ -0,0 +1,329 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Yes, kind of, in this layout font sizes are increased a lot more than font sizes in `dimens`. Increasing sizes in `dimens` even more could look weird on other screens while on tablet.